tests/components/line.test.js

```diff
@@ -0,0 +1,80 @@
+/* global assert, process, setup, suite, test */
+var entityFactory = require('../helpers').entityFactory;
+
+suite('line', function () {
+  setup(function (done) {
+    var el = this.el = entityFactory();
```

Comment:
Not a big deal, but for future tests, I'd move `var el;` up a level and use that so we don't need to do `var el = this.el`. And also create a `var component;` and store the instance in there so we don't have to do `el.components.X`